package.json

```diff
@@ -198,8 +199,7 @@
     "ts-morph": "^9.1.0",
     "ts-node": "^10.6.0",
     "typescript": "^4.7.2",
-    "webpack-cli": "^4.10.0",
     "webpack": "^5.73.0",
-    "@babel/runtime": "^7.18.9"
```

Comment:
Is the removal of `@babel/runtime` really related to this PR?
Please double check with @WeizhengSap , who added it when upgrading to Angular 14 https://github.com/SAP/spartacus/blame/6c9b4ae3c40837158ba213dccb6caf793a7ad2d0/package.json#L202

Reply:
You'll find its not removed. But rather just moving itself to another position in the `package.json`. ;)

Reply:
Exactly - not removed, just moved :)

Reply:
I'm sorry. My bad!